shared/teams/new-team/index.js

```diff
@@ -6,6 +6,32 @@ import {globalColors, globalMargins, globalStyles} from '../../styles'
 
 import type {Props} from './'
 
+const validTeamname = (s: string): boolean => {
+  // This logic is copied from go/protocol/keybase1/extras.go.
+  if (s.length < 2 || s.length > 16) {
```

Comment:
I believe this should only check the length of strings between periods, since an entire teamname can be >16chars even though each subteam name is <16chars

Reply:
You're right! Done.